.kokoro/build.sh

```diff
@@ -71,12 +71,12 @@ integration)
     ;;
 graalvm)
     # Run Unit and Integration Tests with Native Image
-    mvn -B ${INTEGRATION_TEST_ARGS} -ntp -Pnative test
+    mvn -B ${INTEGRATION_TEST_ARGS} -ntp -Pnative integration-test
     RETURN_CODE=$?
     ;;
 graalvm17)
     # Run Unit and Integration Tests with Native Image
-    mvn -B ${INTEGRATION_TEST_ARGS} -ntp -Pnative test
+    mvn -B ${INTEGRATION_TEST_ARGS} -ntp -Pnative integration-test
```

Comment:
I agree with @mpeddada1 's concerns, but want to also add as a side-note:

`mvn integration-test` should rarely be directly invoked. Instead, `mvn post-integration-test` or `mvn verify` should be preferred depending on the situation. See the [Maven lifecycle documentation](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html) for additional information.

In this specific case of `-Pnative`, the configurations become unique due to the behavior of the native image plugin, and we should be using the `mvn test` goal - not `verify` or `post-integration-test`.

Reply:
I changed it to `integration-test` because it is a limitation in maven, please see the [reference](https://graalvm.github.io/native-build-tools/latest/maven-plugin.html#testing-support)